.github/workflows/linux-ci.yml

```diff
@@ -82,11 +82,17 @@ jobs:
           [[ ${{ matrix.build_static }} == "true" ]] && buildtype=static || buildtype=
           platform_str=`python3 tools/hsf_get_platform.py -b $buildtype`
           echo "platform_string=${platform_str}" >> $GITHUB_ENV
+      - name: Generate safe branch name
+        # Replace forward slash in branch name head_ref, such as stable/1.1 
+        # because slash is not allowed in artifact name.
+        run: |
+          head_ref=${{ github.head_ref }}
+          echo "safe_head_ref=${head_ref//\//-}" >> $GITHUB_ENV
```

Comment:
This seems to already be working fine already, I didn't quite understand the need for it.

Reply:
True, I assumed the Linux ci would [fail in the same way as the windows ci](https://github.com/jhmgoossens/CoinUtils/actions/runs/4841032708) because of the message in the error, but I didn’t verify.